src/SFA.DAS.EAS.Employer_Account.Database/StoredProcedures/GetAccountStats.sql

```diff
@@ -5,6 +5,8 @@ AS
 
 	SELECT
 		(SELECT @accountId) AS AccountId,
-		(SELECT COUNT(1) FROM [employer_account].[AccountHistory] WHERE AccountId = @accountId) AS PayeSchemeCount,
-		(SELECT COUNT(1) FROM [employer_account].[AccountLegalEntity] WHERE AccountId = @accountId) AS OrganisationCount,
-		(SELECT COUNT(1) FROM [employer_account].[Membership] WHERE AccountId = @accountId) AS TeamMemberCount
\ No newline at end of file
+		(SELECT COUNT(1) FROM [employer_account].[AccountHistory] WHERE AccountId = @accountId AND RemovedDate IS NULL) AS PayeSchemeCount,
+		(SELECT COUNT(1) FROM [employer_account].[AccountLegalEntity] WHERE AccountId = @accountId AND Deleted IS NULL) AS OrganisationCount,
+		(SELECT COUNT(1) FROM [employer_account].[Membership] WHERE AccountId = @accountId) AS TeamMemberCount,
+		(SELECT COUNT(1) FROM [employer_account].[Invitation] WHERE AccountId = @accountId AND Status = 1) AS TeamMembersInvited,
+		(SELECT COUNT(1) FROM [employer_account].[AccountLegalEntity] WHERE AccountId = @accountId AND SignedAgreementId IS NOT NULL AND Deleted IS NULL) AS SignedAgreementCount
```

Comment:
Don't think we need to pull SignedAgreementCount in stats

Reply:
Removed SignedAgreementCount from stats